[Pages] Clarify vercel migration guide (#21605)
cloudflare/cloudflare-docs: src/content/docs/pages/migrations/migrating-from-vercel/index.mdx

## Add a custom domain

Next, connect a [custom domain](/pages/configuration/custom-domains/) to your Pages project. This domain should be the same one as your currently deployed Vercel application.

### Change domain nameservers

In most cases, you will want to [add your domain to Cloudflare](/dns/zone-setups/full-setup/setup/).

This does involve changing your domain nameservers, but simplifies your Pages setup and allows you to use an apex domain for your project (like `example.com`).

If you want to take a different approach, read more about [custom domains](/pages/configuration/custom-domains/).

### Set up custom domain

<Render file="custom-domain-steps" />

The next steps vary based on if you [added your domain to Cloudflare](#change-domain-nameservers):

- **Added to Cloudflare**: Cloudflare will set everything up for you automatically and your domain will move to an `Active` status.
- **Not added to Cloudflare**: You need to [update some DNS records](/pages/configuration/custom-domains/#add-a-custom-subdomain) at your DNS provider to finish your setup.

## Delete your Vercel app

> After Pages finishes the initial deploy of your site and you have your custom domain set up, delete the Vercel application to start sending requests to Cloudflare Pages.

Author: ```suggestion
Once your custom domain is set up and sending requests to Cloudflare Pages, you can safely delete your Vercel application.
```
